src/components/GrantRequest/GrantRequestFundingSection.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from 'react'
+
+import Label from 'decentraland-gatsby/dist/components/Form/Label'
+import Paragraph from 'decentraland-gatsby/dist/components/Text/Paragraph'
+import useEditor, { assert, createValidator } from 'decentraland-gatsby/dist/hooks/useEditor'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Field } from 'decentraland-ui/dist/components/Field/Field'
+
+import { isValidGrantBudget } from '../../entities/Grant/utils'
+import { asNumber, userModifiedForm } from '../../entities/Proposal/utils'
+import { ContentSection } from '../Layout/ContentLayout'
+
+import { GrantRequestFundingSchema } from './GrantRequestSchema'
+import GrantRequestSection from './GrantRequestSection'
+
+export type GrantRequestFundingState = {
+  funding: string | number
+}
+
+export const INITIAL_GRANT_REQUEST_FUNDING_STATE: GrantRequestFundingState = { funding: '' }
+
+const schema = GrantRequestFundingSchema
```

Comment:
I think you can remove this variable and use `GrantRequestFundingSchema` directly. `GrantRequestFundingSchema.funding.minimum`